convert2rhel/redhatrelease.py

```diff
@@ -92,7 +99,19 @@ def _write_altered_yum_conf(self):
     def get_yum_conf_filepath():
         return YumConf._yum_conf_path
 
+    @staticmethod
+    def _is_modified():
```

Comment:
why are you doing it protected?

Reply:
I suppose because I suggested it under https://github.com/oamg/convert2rhel/pull/189#discussion_r583114922.
You're right there's no reason to have this method protected. Adam, you can change it to:
```suggestion
    def is_modified():
```